tests/e2e/cluster_utils.py

```diff
@@ -74,20 +87,34 @@ def get_ols_url(route_name: str) -> str:
         raise Exception("Error getting route hostname") from e
 
 
+def get_pods() -> list[str]:
+    """Get the names of all pods in the cluster."""
```

Comment:
this should take a namespace arg and pass it to `oc`, otherwise you don't actually know which namespace you're getting the pods for (it will just be whatever the current NS happens to be)

you can default the arg value to `openshift-lightspeed`